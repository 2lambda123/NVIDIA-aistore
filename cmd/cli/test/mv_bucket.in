ais gen-shards --bucket $BUCKET_1 --ext .tar --template="tmp/shard-{0..9}" --fcount 1 --fsize 1KB --conc 1 --cleanup // IGNORE
ais mv bucket $BUCKET_1/ $BUCKET_2/ --wait
ais mv bucket $BUCKET_2/ $BUCKET_1/
ais gen-shards --bucket $BUCKET_3 --ext .tar --template="tmp/shard-{0..9}" --fcount 1 --fsize 1KB --conc 1 --cleanup // IGNORE
ais mv bucket $BUCKET_1/ $BUCKET_3/ // FAIL "bucket "ais://$BUCKET_3" already exists"
ais mv bucket $BUCKET_1/ $BUCKET_1/ // FAIL "cannot mv "ais://$BUCKET_1" as "ais://$BUCKET_1""

ais rm bucket $BUCKET_1
ais rm bucket $BUCKET_3

ais mv bucket az://bucket new_bucket // FAIL "bucket "azure://bucket" does not exist"
