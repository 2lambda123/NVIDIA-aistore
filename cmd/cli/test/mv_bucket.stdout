Renaming bucket ("ais://$BUCKET_1" => "ais://$BUCKET_2")...
Done.
Renaming bucket ("ais://$BUCKET_2" => "ais://$BUCKET_1") is in progress.
To check the status, run: ais show job xaction move_bck ais://$BUCKET_1
